roles/ovirt-repositories/README.md

```diff
@@ -0,0 +1,68 @@
+
+oVirt Repositories
+==================
+
+The `ovirt-repositories` role is used to set the repositories required for oVirt engine installation.
+By default it copies content of /etc/yum.repos.d/ to /tmp/repo-backup-{{timestamp}}.
+
+
+Requirements
+------------
+
+ * oVirt Python SDK version 4
+ * Ansible version 2.4
+
+Role Variables
+--------------
+
+| Name                                       | Default value         |  Description                              |
+|--------------------------------------------|-----------------------|-------------------------------------------|
+| ovirt_repositories_ovirt_rpm_repo          | UNDEF                 | URL of RPM package with repository files. |
+| ovirt_repositories_use_subscription_manager| False                 | If true it will use repos from subscription manager. |
+| ovirt_repositories_rh_username             | UNDEF                 | Username to use for subscription manager. |
+| ovirt_repositories_rh_password             | UNDEF                 | Password to use for subscription manager. |
+| ovirt_repositories_pool_ids                | UNDEF                 | List of pools ids to subscribe to. |
+| ovirt_repositories_repos_backup_path       | /tmp/repo-backup-{{timestamp}} | Directory to backup the original repositories configuration
+
+
+Dependencies
+------------
+
+No.
+
+Example Playbook
+----------------
+
+```yaml
+---
+- name:  install RPM with repository files
+  hosts: localhost
+
+  vars:
+    ovirt_reposotories_ovirt_rpm_repo: http://resources.ovirt.org/pub/yum-repo/ovirt-master-release.rpm
+
+  roles:
+    - role: ovirt-repositories
+
+# It's a good practice to encrypt your credentials with ansible vault
+- include_vars: credentials.yml
```

Comment:
Can you please check how we use `passwords.yml` in all other repos? Let's be consistent.